### ISSUE TITLE
Jurisdiction link resolves to incorrect page

### ISSUE THREAD

**Issue by kereid** [aodn/aodn-portal#2207]
App version: 4.23.2
            Build date: Fri May 20 14:21:32 AEST 2016
            Build number: 54
            Git ref: f3b156f7343f0f766f5f86aa77dcb99d4fba32b8

Steps to reproduce:

2.5cc licence AODN controlled: Add any of RJ satellite collections to portal proceed to step 3 open download popup and click Jurisdiction link

2.5 and 3.0cc licence external contributors: Add any RLS collection and proceed to step 3 open download popup and click Jurisdiction link

4.0cc licence AODN controlled: Add any IMOS collection on portal such as Argo proceed to step 3 open download popup and click Jurisdiction link

What does happen:
2.5cc and 3cc (all): link goes to someones blog page on cc site 

4.0cc: link is broken - 404 error

What should happen:
2.5cc and 3.0cc (all): Jurisdiction link goes to appropriate page on cc site @atkinsn will fix this with upgrade to Attribution 4.0 on AODN controlled metadata and notify external parties for remaining 2.5cc and 3.0

4.0 resolves to appropriate jurisdiction page which will require bulk metadata update, is this a backlog item @pblain?

**Comment by kereid** [comment]
Jurisdiction for 4.0 link should be 

https://wiki.creativecommons.org/wiki/CC_Affiliate_Network

**Comment by atkinsn** [comment]
2.5cc (AODN controlled) - I have fixed by upgrading to 4.0cc (see below)

2.5cc and 3.0cc (external) - The links haven't been fixed along with the resumption of the correct link from the 4.0cc records. I will let Emma know that this will need to be fixed, either by editing link, or upgrading where appropriate the licence to 4.0.

So link on 4.0cc has been fixed, after @kereid alerted the website that one of their links wasn't resolving.
